docs/content/concepts/layer/control/catalog/sp800-53rev5-example.md

```diff
@@ -0,0 +1,340 @@
+---
+title: SP800-53 Control Example
+heading: "NIST SP 800-53 rev 5 Annotated Example"
+weight: 30
+sidenav:
+  title: SP800-53 Control Example
+toc:
+  enabled: true
+---
+
+
+{{<callout>}}**Please note:**
+
+- The following commentary uses and shows XML syntax. A later revision will offer JSON and YAML translations.
+- Full versions of the NIST SP 800-53 Rev. 5 catalog are available in OSCAL [XML]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml), [JSON]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json), and [YAML]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/yaml/NIST_SP-800-53_rev5_catalog.yaml) formats in the OSCAL content GitHub repository. These examples also include assessment and objective content from NIST SP 800-53A Rev. 5, *not shown here*.
```

Comment:
Please spell out "revision".
```suggestion
- Full versions of the NIST SP 800-53 Revision 5 catalog are available in OSCAL [XML]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml), [JSON]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json), and [YAML]({{< param "contentRepoPath" >}}/nist.gov/SP800-53/rev5/yaml/NIST_SP-800-53_rev5_catalog.yaml) formats in the OSCAL content GitHub repository. These examples also include assessment and objective content from NIST SP 800-53A Revision 5, *not shown here*.
```

Reply:
@wendellpiez Can you make this adjustment in the other places where this appears?

Reply:
Will have to do that in post.